python-package/lightgbm/sklearn.py

```diff
@@ -791,7 +791,7 @@ def fit(self, X, y,
                     categorical_feature=categorical_feature, callbacks=callbacks, init_model=init_model)
         return self
 
-    _base_doc = LGBMModel.fit.__doc__
+    _base_doc = LGBMModel.fit.__doc__ or ""
     _base_doc = (_base_doc[:_base_doc.find('group :')]
```

Comment:
can you please revert these `_base_doc` fixes? Another contributor is already helping with them on #3891

Reply:
done